### DISCUSSION TITLE
Remove the "Off" option for security panel

### DISCUSSION THREAD

**Discussion by danielmtnz** [dgreif/ring#1026] [General]
I use Siri to set the alarm at night, however, I would like to disable the capability of disarming it from the Home app. This is because I have iPads on the walls and the Home app would allow anyone in the house to disable it without entering a code. Seems like I should be able to do this by modifying some code in security-panel.js? 

Got this working before by commenting out some lines, but the Off option was still there, it just ignored it when it was selected. However, the way I did it, the Home UI would keep spinning with the wrong status. 

Thank you.

**Comment by dgreif** [comment]
As you experienced, not having an "off" option could lead to some weird states in HomeKit. This isn't a feature I plan to support, but you are welcome to fork the repo or update the code locally to make it work for your use case
